integrations.go

```diff
@@ -197,6 +219,12 @@ func (client *Client) CreateIntegrationAWS(awsAccount *IntegrationAWSAccount) (*
 	return &out, nil
 }
 
+// UpdateIntegrationAWS updates an already existing AWS Account in the AWS Integration
+func (client *Client) UpdateIntegrationAWS(awsAccount *IntegrationAWSAccount) error {
+	additionalParameters := "account_id=" + *awsAccount.AccountID + "&role_name=" + *awsAccount.RoleName
```

Comment:
I'd much rather see usage of `url.Values` here: https://golang.org/pkg/net/url/#Values which will give us safety guarantee and is the canonical way of constructing query strings in go.

Reply:
That is indeed cleaner that what I did in the first place. It's using it now, thank you for the tip!